docs/install.rst

```diff
@@ -70,6 +70,10 @@ need to install Python 3 with virtualenv in your system as well.
 
         sudo apt-get install redis-server
 
+.. note::
+
+   Some unit tests depend on Hg (mercurial).
+   Be sure that Hg is `installed <https://www.mercurial-scm.org/wiki/Download>`_ if you want to see all the tests pass.
```

Comment:
The mercurial change was not supposed to be in there...
I am not having initial success with rebasing. Its not something I do often :/ I'll continue to work on it.